build.gradle

```diff
@@ -60,6 +60,33 @@ def distributions = [
                          "version": "2.0.0~b5",
                          "release": '1'
                         ],
+                        ["dist": "openhab2-release",
```

Comment:
I would prefer to call it openhab2-2.0.0 or openhab-2_0_0/openhab-200 if this does not work. So we are prepared for a 2.1.0 release.